cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -120,14 +118,15 @@ func (cm *certManagerClient) EnsureWebhook() error {
 	}
 
 	// installs the web-hook
+	createCertManagerBackoff := wait.Backoff{Duration: 500 * time.Millisecond, Factor: 1.5, Steps: 10}
```

Comment:
May be better to create a newBackoff func that returns these, as the magic values are repeated in multiple files. Will allow you tweak the retries in one place. CAPA has https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/services/wait/wait.go#L34